src/custom/components/CowSubsidyModal/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { useCallback } from 'react'
+import {
+  ConfirmationModalContent,
+  ConfirmationModalContentProps,
+  ConfirmationModalProps,
+} from 'components/TransactionConfirmationModal'
+import { useActiveWeb3React } from 'hooks/web3'
+import { GpModal } from 'components/Modal'
+import { AutoColumn } from 'components/SearchModal/CommonBases'
+import { Text } from 'rebass'
+
+import { useTokenBalance } from 'state/wallet/hooks'
+import { V_COW } from 'constants/tokens'
+
+import Row from 'components/Row'
+import { ExternalLink } from 'components/Link'
+
+import { COW_SUBSIDY, SUBSIDY_INFO_MESSAGE } from 'constants/subsidy'
+import CowBalance, { CowBalanceProps } from '../CowBalance'
+import SubsidyTable from './SubsidyTable'
+
+const CowSubsidyInfo = ({ account, balance }: CowBalanceProps) => (
+  <AutoColumn style={{ marginTop: 20, padding: '2rem 0' }} gap="24px" justify="center">
+    <Text fontWeight={500} fontSize={16} style={{ textAlign: 'center', width: '85%', wordBreak: 'break-word' }}>
+      {SUBSIDY_INFO_MESSAGE}
+    </Text>
+    {/* VCOW LOGO */}
+    <CowBalance account={account} balance={balance} />
+    <SubsidyTable data={COW_SUBSIDY} />
```

Comment:
you should pass the balance too, right? Just to highlight the row

Reply:
incoming in another pr these were split